client/tests/pw/helpers/utils.ts

```diff
@@ -45,6 +45,11 @@ export async function fillIonInput(ionInput: Locator, text: string): Promise<voi
   await input.blur();
 }
 
+export async function clearIonInput(ionInput: Locator): Promise<void> {
```

Comment:
An input can be cleared by using `fill('')`, we don't really need another function. Or if you really feel like we need it, it could just be defined as
`export const clearIonInput = async (ionInput: Locator) => await fillIonInput(ionInput, '');`